ios/RocketChatRN.xcodeproj/project.pbxproj

```diff
@@ -179,10 +379,10 @@
 			name = Pods;
 			sourceTree = "<group>";
 		};
-		607D60ED1F325B7D00F639C4 /* Products */ = {
+		7AAA749723043AD300F1ADE9 /* Products */ = {
 			isa = PBXGroup;
 			children = (
-				607D61161F325B7E00F639C4 /* libRealmReact.a */,
+				7AAA749B23043AD300F1ADE9 /* libWatermelonDB.a */,
```

Comment:
Is this correct?
Realm is removed and Watermelon is already merged.
Why is this changing on this PR?